SimPEG/Examples/Downloads/SimPEGtemp/LdM_grav_obs.grv

```diff
@@ -0,0 +1,192 @@
+191				
```

Comment:
maybe an option at the top of the script to make it clear
`cleanAfterRun = True` ?